sdk/keyvault/azure-security-keyvault-jca/src/main/java/com/azure/security/keyvault/jca/KeyVaultLoadStoreParameter.java

```diff
@@ -74,6 +74,22 @@ public KeyVaultLoadStoreParameter(String uri, String tenantId, String clientId,
         this.managedIdentity = null;
     }
 
+    /**
+     * Constructor.
+     *
+     * @param uri the Azure Key Vault URI.
+     * @param tenantId the tenant ID.
+     * @param clientId the client ID.
+     * @param clientSecret the client secret.
+     * @param managedIdentity the managedIdentity.
+     */
+    public KeyVaultLoadStoreParameter(String uri, String tenantId, String clientId, String clientSecret, String managedIdentity) {
```

Comment:
Could you please make other constructor call this one?

Reply:
There are still 2 constructors not calling this constructor, could you please update it?

Reply:
My fault, I forgot this, it has been added in the latest commit.